resolwe_bio/processes/plots/bamliquidator.yml

```diff
@@ -0,0 +1,216 @@
+  category: analyses
+  persistence: CACHED
+  description: >
+    Set of tools for analyzing the density of short DNA sequence read alignments in the BAM file format.
+  input:
+    - name: analysis_type
+      label: Analysis type
+      type: basic:string
+      default: bin
+      choices:
+        - label: Bin mode
+          value: bin
+        - label: Region mode
+          value: region
+        - label: BED mode
+          value: bed
+    - name: bam
+      label: BAM File
+      type: list:data:alignment:bam
+    - name: cell_type
+      label: Cell type
+      type: basic:string
+      default: cell_type
+    - name: bin_size
+      label: Bin size
+      type: basic:integer
+      hidden: "analysis_type != 'bin'"
+      required: False
+      description: >
+        Number of base pairs in each bin. The smaller the bin size the longer the runtime and the larger the data files.
+        Default is 100000.
+    - name: regions_gtf
+      label: Region gff file / Annotation file (.gff|.gtf)
+      type: data:annotation:gtf
+      hidden: "analysis_type != 'region'"
+      required: False
+    - name: regions_bed
+      label: Region bed file / Annotation file (.bed)
+      type: data:bed
+      hidden: "analysis_type != 'bed'"
+      required: False
+    - name: extension
+      label: Extension
+      type: basic:integer
+      default: 200
+      description: >
+        Extends reads by number of bp
+    - name: sense
+      label: Mapping strand to gff file
+      type: basic:string
+      default: "."
+      choices:
+        - label: Forward
+          value: +
+        - label: Reverse
+          value: "-"
+        - label: Both
+          value: "."
+    - name: skip_plot
+      label: Skip plot
+      type:  basic:boolean
+      required: False
+    - name: black_list
+      label: Black list
+      type: list:basic:string
+      required: False
+      description: >
+        One or more chromosome patterns to skip during bin liquidation. Default is to skip any
+        chromosomes that contain any of the following substrings chrUn _random Zv9_ _hap.
+    - name: threads
+      label: Threads
+      type: basic:integer
+      default: 1
+      description: >
+        Number of threads to run concurrently during liquidation.
+  output:
+    - name: analysis_type
+      label: Analysis type
+      type: basic:string
+      hidden: true
+    - name: output_dir
+      label: Output directory
+      type: basic:file
+    - name: counts
+      label: Counts HDF5 file
+      type: basic:file
+    - name: matrix
+      label: Matrix file
+      type: basic:file
+      required: False
+      hidden: "analysis_type != 'region'"
+    - name: summary
+      label: Summary file
+      type: list:basic:file:html
+      required: False
+      hidden: "analysis_type != 'bin'"
+  run:
+    runtime: polyglot
+    language: bash
+    program: |
+
+
+      re-save analysis_type {{ analysis_type }}
+
+      {% if regions_gtf and regions_bed %}
```

Comment:
{% if analysis_type == "bin" and not bin_size %} is not needed, since default bin size is set.

Is any basic difference if is written in bash or in Django?